src/components/signup/__tests__/CheckSaveW3LoginToken.js

```diff
@@ -1,6 +1,11 @@
 import userStorage from '../../../lib/gundb/UserStorage'
 
 describe('Check W3 Login Token Save', () => {
+  beforeAll(async () => {
+    jest.setTimeout(15000)
+    await userStorage.ready
```

Comment:
are you don't need to await wallet here too ?

Reply:
@serdiukov-o-nordwhale 
actually waiting just for userStorage.ready is enough as it uses the wallet.ready promise inside.